src/main/scala/org/broadinstitute/dsde/firecloud/service/MethodsService.scala

```diff
@@ -17,10 +17,19 @@ trait MethodsService extends HttpService with FireCloudDirectives {
 
   private final val ApiPrefix = "methods"
 
-  val routes = listMethodsRoute
+  val routes = { optionsRoute ~ listMethodsRoute }
```

Comment:
The braces shouldn't be necessary here. Stylistically, I'd leave 'em out - willing to hear otherwise.